app/locale/en.coffee

```diff
@@ -150,6 +150,8 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     want_coco: "Want CodeCombat at your school?"
     educator: "Educator"
     student: "Student"
+    china_contact_phone: '13810906731'
```

Comment:
Either just hardcode these in the template files or have them be a constant somewhere else, maybe app/lib/constants.coffee or something. Don't put them in en.coffee because they don't need translation.

Reply:
Do I have to declare a variable in .coffee file first then I can use that on jade?

Reply:
There are two ways I can think of that you can access a variable from the jade template file.


## 1. If the variable is a property of the view you can access it directly

I.e. State is on the view (in the coffeescript file): https://github.com/codecombat/codecombat/blob/c65326ae5e3fda3637aed2f8e33823134125c4ac/app/views/courses/EnrollmentsView.coffee#L38

Properties on this view can be directly accessed like so: https://github.com/codecombat/codecombat/blob/ce2bca1025f4c15918d714fd97dd82254dde5be5/app/templates/courses/enrollments-view.jade#L42

## 2. Properties can be attached using the `getRenderData` method.

Example of usage: https://github.com/codecombat/codecombat/blob/983b6ac2f33c07b6619f8b1d3fc5a4aaa0386bce/app/views/play/modal/PollModal.coffee#L24